pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -33,28 +36,38 @@ From the list of healthy peers that you retrieve from peer node provider, choose
-
-### State Sync
-You can use **state sync**, a set of configuration settings that allow your node to retrieve a snapshot from the network. If your node falls out of sync, it queries a state sync node for a verified, recent snapshot of the application state. This speeds up the syncing process because you avoid replaying the entire history of the network, instead starting from the network's most recent application state snapshot. To use state sync to back up your full node, follow the instructions for your deployment from a [State Sync](../infrastructure_providers-network/resources.mdx#state-sync-service) service.
+Save the `addrbook.json` file in your `/.dydxprotocol/config` directory.
 
-## Optimize Pruning Settings
+### Configure a Pruning Strategy
 In general, dYdX recommends the following pruning setting, configured in your `app.toml` file:
 
 ```bash
 # app.toml
 pruning = "everything" # 2 latest states will be kept; pruning at 10 block intervals
 ```
 
-However, if you want to use your node to query historical data, configure a custom pruning strategy to retain more states. Retaining more states increases storage requirements.
\ No newline at end of file
+However, if you want to use your node to query historical data, configure a custom pruning strategy to retain more states. Retaining more states increases storage requirements.
+
+## Prepare to Restore Your Node
+Prepare to restore your node quickly in case it falls out of sync. 
+
+Your full node can fall out of sync with the rest of the network for a variety of reasons, including a bad software upgrade, unexpected node crashes, or human operational error. To re-sync with the network, your full node must replay the history of the network.
+
+You can speed up the re-syncing process significantly by providing your node with a snapshot. A snapshot contains a compressed copy of the application state at the time the snapshot was taken. If your node falls out of sync, a snapshot allows it to recover to that saved state before replaying the rest of the history of the network, saving time.
+
+### Save a Snapshot on Your System
+You can use a snapshot saved on the system on which your node runs to restore your node.
+
+To use a snapshot to back up your full node, install a snapshot for your deployment from a [Snapshot Service](../infrastructure_providers-network/resources.mdx#snapshot-service).
+
+> If you followed the procedure on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), you already have a snapshot installed.
+
+### Configure Your Node's State Sync
+You can use state sync, a configuration setting that allows your node to retrieve a snapshot from the network, to restore your node.
```

Comment:
State sync is a process that automates the state syncing process by automatically fetching snapshots.

I think it's worth reading [this doc](https://blog.cosmos.network/cosmos-sdk-state-sync-guide-99e4cf43be2f) and summarizing it in 1-2 sentences here for a high-level overview of this approach. I found this part to be sufficient:


```
... state sync, which allows a new node to join a network by fetching a snapshot of the application state at a recent height instead of fetching and replaying all historical blocks. This can reduce the time needed to sync with the network from days to minutes.

..., the good news is that the upcoming Cosmos SDK 0.40 release will include automatic support for state sync, and developers only need to enable it in their applications to make use of it. We’ll explain how in this article, but first some background.
```